default.cfg

```diff
@@ -46,7 +46,7 @@ abs-error = true
 
 ## If true then relative round-off error is computed.
 [short: rel]
-rel-error = false
+rel-error = true
```

Comment:
This must be `false` by default: relative error computations are not supported for all examples. So users should enable this option explicitly in their own configuration files.